contracts/cw20-base/src/contract.rs

```diff
@@ -153,7 +167,13 @@ pub fn instantiate(
         MARKETING_INFO.save(deps.storage, &data)?;
     }
 
-    Ok(Response::default())
+    let serialized_balances = serde_json::to_string(initial_balances).unwrap();
```

Comment:
I don't feel the benefit of doing this beats the cost.

Reply:
Will update this code